what4/src/What4/Interface.hs

```diff
@@ -537,7 +537,6 @@ class (IsExpr (SymExpr sym), HashableF (SymExpr sym)) => IsExprBuilder sym where
 
   -- | Equality of boolean values
   eqPred  :: sym -> Pred sym -> Pred sym -> IO (Pred sym)
-  eqPred sym x y = notPred sym =<< xorPred sym x y
```

Comment:
What's the problem with this default implementation?

Reply:
Nothing, really, but `xor` is defined as the negation of equality now, and eqPred is actually the primitive that is defined.  The only real problem is that the definition is misleading